library/src/engine/engine.ts

```diff
@@ -875,15 +875,15 @@ function applyAttributePlugin(
   attrKey: string,
   value: string,
 ): void {
-  const rawKey = camel(alias ? attrKey.slice(alias.length) : attrKey)
+  const rawKey = alias ? lowerFirst(attrKey.slice(camel(alias).length)) : attrKey
   const plugin = plugins.find((_, i) => pluginRegexs[i].test(rawKey))
   if (plugin) {
     // Extract the key and modifiers
     let [key, ...rawModifiers] = rawKey.slice(plugin.name.length).split(/__+/)
 
     const hasKey = !!key
     if (hasKey) {
-      key = camel(key)
```

Comment:
Same as above. Since the key follows a dash that isn't meant to capitalize the next letter, but separate the key from the plugin name, we just need to lower case the first letter.